apps/api/src/domain/models/term/commands/elicit-term-from-prompt/term-elicited.from.prompt.ts

```diff
@@ -7,4 +8,8 @@ export type TermElicitedFromPromptPayload = ElicitTermFromPrompt;
 @CoscradEvent('TERM_ELICITED_FROM_PROMPT')
 export class TermElicitedFromPrompt extends BaseEvent<TermElicitedFromPromptPayload> {
     readonly type = 'TERM_ELICITED_FROM_PROMPT';
+
+    override buildAttributionStatement(): string {
+        return `${formatLanguageCode(this.payload.languageCode)} term provided by: `;
```

Comment:
I know it was a bit awkward, but I'm wondering if this should say something like `Chilcotin term added to system by: `.  The word "provided" suggests that this action is being done by the speaker who knows and is providing the term (ie, an attribution), where this contribution statement could be reffering to a non-speaker admin or hire who is doing the data import or data entry.  Could be a bit misleading?

Reply:
Ok this discussion was moved to discord, where we found a temporary work around. In the long run, we want a separate property for data entry (userId) compared to contributions.